.github/dependabot.yml

```diff
@@ -12,8 +12,8 @@ updates:
           - dependency-name: "@khanacademy/eslint-plugin"
           - dependency-name: "@khanacademy/wonder-blocks-*"
```

Comment:
would it be useful adding `Khan/wonder-blocks` for reviewing these changes? I'm not sure if that would be a good signal for us to know when we can update the `webapp` repo to have synced deps? or do you recommend something different?

How do you think the process should work now that dependabot is connected for these type of updates?

Reply:
I can add that team, for sure.  If it gets noisy we can change it